lib/components/map/connected-geojson-layer.js

```diff
@@ -0,0 +1,79 @@
+import {
+  Styled as BaseMapStyled,
+  MarkerWithPopup
+} from '@opentripplanner/base-map'
+import { connect } from 'react-redux'
+import FromToLocationPicker from '@opentripplanner/from-to-location-picker'
+import React, { useEffect, useState } from 'react'
+
+import { setLocation } from '../../actions/map'
+
+const GeoJSONOverlay = (props) => {
+  const { setLocation, url } = props
+
+  const [locations, setLocations] = useState([])
+  useEffect(() => {
+    async function downloadLocations() {
+      const json = await (await fetch(url)).json()
+      setLocations(json)
+    }
+    if (url) downloadLocations()
+  }, [])
+
+  return (
+    <>
+      {locations?.features?.map((feature, k) => {
+        const { geometry, properties } = feature
+        if (!geometry || !geometry.coordinates) return null
+        return (
+          <MarkerWithPopup
+            key={k}
+            popupContents={
+              <BaseMapStyled.MapOverlayPopup>
+                {properties.Name && (
+                  <BaseMapStyled.PopupTitle>
+                    {properties.Name}
+                  </BaseMapStyled.PopupTitle>
+                )}
+                <BaseMapStyled.PopupRow>
+                  {properties.popupContent && (
+                    <div>{properties.popupContent}</div>
+                  )}
+                  {properties.Address && (
+                    <div>
+                      {properties.Address}, {properties.Zip}, {properties.City},{' '}
+                      {properties.State}
+                    </div>
+                  )}
+                  {properties.Phone && <div>{properties.Phone}</div>}
+                  <FromToLocationPicker
+                    label
+                    location={{
+                      lat: geometry.coordinates[1],
+                      lon: geometry.coordinates[0],
+                      name: properties.Name
+                    }}
+                    setLocation={setLocation}
+                  />
+                </BaseMapStyled.PopupRow>
+              </BaseMapStyled.MapOverlayPopup>
+            }
+            popupProps={{ offset: 10 }}
+            position={[geometry.coordinates[1], geometry.coordinates[0]]}
+          >
+            <img
+              alt={properties.Name}
+              src={properties.icon}
+              style={{ height: 40 }}
```

Comment:
This is coming across as very tall!!

Reply:
I think a `className` should still be set on the `<img>` tag so that the configs can style the images if needed.